static/js/impala/listing.js

```diff
@@ -10,7 +10,11 @@ $(function() {
             // L10n: {0} is an app name.
             var msg = format(gettext('This theme is incompatible with your version of {0}'),
                              [z.appName]);
-            $this.append(format('<span class="notavail">{0}</span>', msg));
+            if ($('body').hasClass('restyle')) {
+              $this.append(format('<button class="disabled concealed notavail">{0}</button>', msg));
```

Comment:
I might be missing something here - but doesn't this add a button and then by virtue of the css class concealed it's then removed via`display: none`? Does it need to be added in the first place?